<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Session</class>
 <widget class="QWidget" name="Session">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#credentialsContainer {
	background-color: rgb(21, 8, 39);
	border-radius: 14px;
	border: 2px solid rgb(25, 13, 43);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="credentialsContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>260</width>
       <height>168</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  background-color: rgb(38, 143, 71);
  border: 1px solid transparent;
  border-radius: 5px;
  color: #FFFFFF;
  font-family: &quot;Inter var&quot;,ui-sans-serif,system-ui,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;;
  font-size: 14px;
  font-weight: 600;
	padding-top: 5px;
	padding-bottom: 5px;
}

QPushButton:hover {
  background-color: rgb(97, 255, 94);
}

QPushButton::disabled {
	background-color: rgb(116, 116, 116);
	color: rgb(190,190,190);
}
QLineEdit {
	border: 2px solid rgb(63, 17, 75);
	background-color: rgb(40, 18, 58);
	font-size: 14px;
	color:white;
	padding-top: 2px;
	padding-bottom: 2px;
	font-weight: 600;
}</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="leftMargin">
       <number>32</number>
      </property>
      <property name="rightMargin">
       <number>32</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <item row="1" column="1">
       <widget class="QLineEdit" name="loginEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="passEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hasło</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font-size: 18px;
font-weight: 600;
color: rgb(255, 216, 255);
margin-bottom:4px;</string>
        </property>
        <property name="text">
         <string>Zaloguj się</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Zaloguj</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
